public/index.html

```diff
@@ -4,11 +4,23 @@
   <link rel="stylesheet" type="text/css" href="styles/interface.css">
   <script src="/socket.io/socket.io.js"></script>
   <script src="scripts/main.js"></script>
+  <script src="http://code.jquery.com/jquery-1.11.0.min.js"></script>
 </head>
 
 <body>
-  <input id="input" autocomplete="off" /><button onClick="sendMessage()">SEND</button>
+
+  
   <ul id='messages'></ul>
 </body>
+<footer>
+  <input class="input" id="input" autocomplete="off" action="sendMessage()" /><button class="button" id="button" onClick="sendMessage()">SEND</button>
```

Comment:
See https://github.com/ZakMiller/TheEmpire/pull/17#issuecomment-276235495, I wonder if it'd be better to handle our event listeners outside of the html tags. I feel that it's more clean to keep the html as just structure and let the scripts add listeners and whatnot, but I'm curious what you all think. Any opinions?